src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1083,6 +1082,18 @@ so that we can apply CSS-filters to change the arrow color in themes */
 	padding: 0 20px 20px 17px;
 }
 
+.chip {
```

Comment:
Could you extract the common parts of `.chip` and `.stab` into `.chip, .stab { /* … */ }`?